### PR TITLE
[PLAT-5091] Support for operations on PMI annotations

### PR DESCRIPTION
**Pull request by amvertex**
## Summary
This PR includes support for operating on PMI annotations within the new scene.elements() method.

## Test Plan
Verify scene item interactions continue to work as expected

## Release Notes
- Includes support for operating on PMI annotations

## Possible Regressions
Interacting with scene items

## Dependencies
None

### CODE REVIEW

File: packages/viewer/src/lib/scenes/scene.ts [hunk 1]
```diff
@@ -559,17 +569,139 @@ export class SceneItemOperationsBuilder
...
+   *
+   * @example
+   * ```typescript
+   * const viewer = document.querySelector('vertex-viewer');
+   * const scene = await viewer.scene();
+   *
+   * // Hide the item with the `item-uuid` ID
+   * await scene.elements((op) => [
+   *   op.items.where((q) => q.withItemId('item-uuid')).hide(),
+   * ]).execute();
```
Comment (jdm717) [inline packages/viewer/src/lib/scenes/scene.ts:610]: Looks like these examples still reference the `items` equivalent, could we update them to the `annotations` operation?

**Review by jdm717 — COMMENTED** [review]
LGTM! Just had a couple thoughts

File: packages/viewer/src/lib/scenes/scene.ts (hunk 1 above)
Comment (amvertex) [reply packages/viewer/src/lib/scenes/scene.ts]: Good catch, updated!

File: packages/viewer/src/lib/scenes/mapper.ts [hunk 2]
```diff
@@ -167,15 +167,53 @@ export function buildQueryExpression(
...
+          first: buildAnnotationQueryExpression(query.expressions[0], context),
+          second: buildAnnotationQueryExpression(query.expressions[1], context),
+        },
+      };
+    case 'or':
+      return {
+        or: {
+          first: buildAnnotationQueryExpression(query.expressions[0], context),
+          second: buildAnnotationQueryExpression(query.expressions[1], context),
+        },
```
[the remark this answers is not in the document]
Comment (amvertex) [reply packages/viewer/src/lib/scenes/mapper.ts:214]: Good catch. What do you think of this?

**Review by jdm717 — APPROVED** [review]
LGTM!

**Pull request merged**